modules/ecs/ipfs/main.tf

```diff
@@ -49,24 +49,27 @@ resource "aws_ecs_task_definition" "main" {
       memory            = var.ecs_memory
       region            = var.aws_region
 
-      ceramic_network     = var.ceramic_network
-      enable_api          = true
-      enable_gateway      = true
-      enable_pubsub       = var.enable_pubsub
+      ceramic_network = var.ceramic_network
+      enable_api      = true
+      enable_gateway  = true
+      enable_pubsub   = var.enable_pubsub
 
       api_port              = local.api_port
       gateway_port          = local.gateway_port
       healthcheck_port      = local.healthcheck_port
-      swarm_port            = local.swarm_port
+      swarm_tcp_port        = local.swarm_tcp_port
       swarm_ws_port         = local.swarm_ws_port
+      swarm_wss_port        = local.swarm_wss_port
+      swarm_config_tcp_port = local.swarm_tcp_port
```

Comment:
Can probably eliminate `swarm_config_tcp_port` and `swarm_config_ws_port` and just do [this](https://github.com/ceramicnetwork/terraform-aws-ceramic/pull/12/files#r829519412)?